assets/charts/components/istio-operator/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: istio-operator
+version: 1.6.0
```

Comment:
Should we update it to 1.7.0? Just asking.

Reply:
I'm personally fine with merging as it is, as the PR is open for a long time already and updating as part of the next release.

Reply:
Yes, I agree with the same